assets/js/modules/analytics-4/datastore/webdatastreams.test.js

```diff
@@ -714,33 +715,39 @@ describe( 'modules/analytics-4 webdatastreams', () => {
 			const accountSummaries = {
 				accountSummaries: [
 					{
-						_id: '123456',
 						propertySummaries: [
-							{ _id: '1122334455' },
-							{ _id: '1122334456' },
-							{ _id: '1122334457' },
+							{ property: 'properties/1122334455' },
```

Comment:
This is necessary because of the populating of ID that we do in the reducer callback for `receiveGetAccountSummaries` which will otherwise overwrite the pre-populated ID field here with `undefined` since the source property is missing. This was the only instance where this was needed so I opted to update the test instead of the populate utilities. It might be a good idea to preserve an existing/pre-populated ID in the future.